runtime/manifest.js

```diff
@@ -549,9 +549,11 @@ ${e.message}
       particleItem.implFile = loader.join(manifest.fileName, particleItem.implFile);
     }
 
-    for (let arg of particleItem.args) {
+    let processArgTypes = args => args.forEach(arg => {
```

Comment:
I think it's better to prefer for/of over forEach unless you have an existing function to pass to it.